website/docs/components/collapse.mdx

```diff
@@ -72,16 +72,58 @@ function Example() {
 }
 ```
 
+### Customizing the transition state styles
+
+The `config` prop is used to style the Transition component's initial, entered, and 
+exiting states. This allows the height, opacity, transform, transition, or any 
+other valid CSS property to be configured for each state.
+
+```jsx
+function Example() {
+  const [show, setShow] = React.useState(false)
```

Comment:
Could you replace this and the related functionality with `useDisclosure`?

Reply:
Yep updated every example in the Collapse docs.